ChangeLog.d/sha256-armce-arm.txt

```diff
@@ -0,0 +1,4 @@
+Features
+    * Support Armv8 Crypto Extension acceleration for SHA-256
+      when compiling for Thumb or 32-bit Arm.
```

Comment:
```suggestion
      when compiling with clang for Thumb or 32-bit Arm.
```

Reply:
It's also supported with GCC, but runtime detection is not.